test-suite/coq-makefile/timing/run.sh

```diff
@@ -74,7 +74,7 @@ echo
 
 for ext in "" .desired; do
     for file in A.v.timing.diff; do
-        cat ${file}${ext} | sed s"/${INFINITY}/${INFINITY_REPLACEMENT}/g" | sed s":|\s*N/A\s*$:| ${INFINITY_REPLACEMENT}:g" | sed s'/[0-9]*\.[0-9]*//g' | sed s'/0//g' | sed s'/  */ /g' | sed s'/ *$//g' | sed s'/+/-/g' | sort > ${file}${ext}.processed
+        cat ${file}${ext} | sed s"/${INFINITY}/${INFINITY_REPLACEMENT}/g" | sed s":|\s*N/A\s*$:| ${INFINITY_REPLACEMENT}:g" | sed s'/[0-9]*\.[0-9]*//g' | sed s'/0//g' | sed s'/^-*$/------/g' | sed s'/  */ /g' | sed s'/ *$//g' | sed s'/+/-/g' | sort > ${file}${ext}.processed
```

Comment:
This is pretty unreadable. How about a cleanup step using an array to gather the sed expressions:
```bash
TOSED_IN=(
    s"/${INFINITY}/${INFINITY_REPLACEMENT}/g"
    s'/[0-9]//g'
    s'/ *$//g'
    s":|\s*N/A\s*$:| ${INFINITY_REPLACEMENT}:g"
    s'/^-*$/------/g'
    s'/  */ /g'
    s'/\(Total.*\)-\(.*\)-/\1+\2+/g'
)

# we need to add -e before each
TOSED=()
for s in "${TOSED_IN[@]}"; do TOSED+=("-e" "$s"); done

for ext in "" .desired; do
    for file in time-of-build-before.log time-of-build-after.log time-of-build-both.log; do
        cat ${file}${ext} | grep -v 'warning: undefined variable' | sed "${TOSED[@]}" > ${file}${ext}.processed
```
(this is for the 1st sed block at line 43, not the one this comment is attached to)
Each expression (line) could have a comment explaining why it's necessary.

Also there are 2 similar looking sed blocks, should they be the same? In which case this PR is missing the first one.

Reply:
Or even using `sed -f`.

Reply:
There should not be enough variation that the first block needs this, but I've added it anyway.  I now use bash arrays and have added comments